src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -616,7 +617,8 @@ export const ComputeModalBase = ({
     const metricsEvent = Utils.cond(
       [(viewMode === 'deleteEnvironment'), () => 'cloudEnvironmentDelete'],
       [(!!existingRuntime), () => 'cloudEnvironmentUpdate'],
-      () => 'cloudEnvironmentCreate'
+      () => 'cloudEnvironmentCreate',
+      [(isCustomImage), () => 'cloudEnvironmentCustomCreate']
```

Comment:
I don't think this is correct. This is a Utils.cond, which means that only one branch at most is triggered. However, a cloud environment can be created **with** a custom image.